cmd/substreams-sink-kv/serve.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/streamingfast/derr"
 	"github.com/streamingfast/shutter"
 	"github.com/streamingfast/substreams-sink-kv/db"
-	pbkv "github.com/streamingfast/substreams-sink-kv/pb/substreams/sink/kv/v1"
+	kvv1 "github.com/streamingfast/substreams-sink-kv/pb/sf/substreams/sink/kv/v1"
```

Comment:
Let's keep `pbkv`, Golang standard is to not put `v1` in versioned package (you just bump the version in the import path).

Reply:
Oh interesting. This is what `buf` produces by default. Happy to change it though. Any idea if `buf` can be configured to follow `golang` standard?

Reply:
Didn't you change `option go_package = "github.com/streamingfast/substreams-sink-kv/pb/sf/substreams/sink/kv/v1;kvv1";`?

Reply:
I know google uses `<short_name>pb` for the package, historically we use `pb<short_name>`, I never seen however the package name in go having a version number in it by default (if you need both v1 and v2, usually the have a suffix).

Reply:
Yeah I changed that because `buf` was generating the package name as `kvv1` and I thought to myself "better be consistent with that everywhere then" (because in some places the import is already named `kvv1`). Happy to go with either of the two though. Wasn't aware of the golang standard and am just a bit surprised that `buf` doesn't follow that by default.

Reply:
Nevermind. Not sure why I thought `buf` generated it as `kvv1` but when I just re-ran it it properly generated it as `package pbkv` /shrug/

Reply:
I'm surprised also because definitely the versioning of Golang module is to just update the import path and leave the package name untouched. Can you tell me what you did first so that it generated `kvv1`?

Let's keep our way of writing the package name for now (so `pbkv`).